nestkernel/simulation_manager.cpp

```diff
@@ -375,6 +376,61 @@ nest::SimulationManager::simulate( Time const& t )
   t_slice_begin_ = timeval();
   t_slice_end_ = timeval();
```

Comment:
But I'll comment it, since it's clearly a mess. The code really needs to be refactored into several classes, splitting the current state of the machinery... but I don't want the job.